css/templates/land-see/page.css

```diff
@@ -514,3 +532,35 @@ h4,
     height: 392px;
   }
 }
+
+/* Popular/Recent/Reccomended Stories */
+
+.land-see-popular-content,
+.land-see-recommended-content,
+.land-see-recent-content {
+  max-width: 850px;
+}
+
+.land-see-popular-content .amp-carousel-button,
```

Comment:
don't you want all the carousels in this theme/template to have these arrows? Remember some one who downloads this template could also be using  something from ampstart.com/components#media. I think you can override all the arrows - what do you think!

Reply:
I do want the arrows to come from the same background SVG CSS for all carousels in the template, yes. The hero carousel arrows are bigger than the popular and recommended ones, however, so I was planning on applying a separate style to those and overriding the `background-size` property. I'm cool with simplifying the CSS here, though, too.